package.nls.de.json

```diff
@@ -1,142 +1,142 @@
 {
-  "cmd.downloadSettings.error.removeExtFail": "Sync: kann Nicht entfernen Sie einige Erweiterungen.",
-  "cmd.downloadSettings.error.unableSave": "Sync: Unable to save extension settings-Datei.",
+  "cmd.downloadSettings.error.removeExtFail": "Sync: Einige Erweiterungen konnten nicht entfernt werden",
+  "cmd.downloadSettings.error.unableSave": "Sync: Die Extension Datei konnte nicht gespeichert werden.",
   "cmd.downloadSettings.info.downloaded": "Sync: Download Abgeschlossen.",
-  "cmd.downloadSettings.info.gotLatestVersion": "Sync: Du hast bereits die neueste version der gespeicherten Einstellungen.",
-  "cmd.downloadSettings.info.readdingOnline": "Sync: Das Lesen Der Einstellungen Online.",
+  "cmd.downloadSettings.info.gotLatestVersion": "Sync: Du hast bereits die neueste Version der gespeicherten Einstellungen.",
+  "cmd.downloadSettings.info.readdingOnline": "Sync: Lese Online Einstellungen.",
   "cmd.downloadSettings.title": "Sync: Download-Einstellungen",
-  "cmd.howSetting.title": "Sync: Wie Konfigurieren",
-  "cmd.otherOptions.customizedSync": "Synchronisieren: Hinzufügen von Benutzerdefinierten Sync Datei-Pfad",
-  "cmd.otherOptions.customizedSync.done": "Sync: {0} registriert worden ist.",
-  "cmd.otherOptions.customizedSync.placeholder": "Geben Sie einen absoluten Pfad für die zusätzlichen hochgeladenen Datei.",
-  "cmd.otherOptions.customizedSync.prompt": "(ex. /Pfad/zu/.eslintrc) werden hoch-und runtergeladen werden in den angegebenen Pfad.",
-  "cmd.otherOptions.downloadCustomFile": "Synchronisieren: Importieren Sie Benutzerdefinierte Sync-Datei in den workspace",
+  "cmd.howSetting.title": "Sync: Wie soll es konfiguriert werden",
```

Comment:
I'm not sure in which content this title is getting shown so maybe the translation is not the best in this situation.

Can you tell me how to trigger this message so I can take a look?